app/views/payment-holds.njk

```diff
@@ -41,13 +45,19 @@
                 <td class="govuk-table__cell">{{ paymentHold.holdCategoryName }}</td>
                 <td class="govuk-table__cell">{{ paymentHold.holdCategorySchemeName }}</td>
                 <td class="govuk-table__cell">{{ paymentHold.dateTimeAdded }}</td>
-                <td class="govuk-table__cell">{{ paymentHold.dateTimeClosed }}</td>
+                <td class="govuk-table__cell">
+                  <form method="POST" action="/remove-payment-hold">
+                    <input type="hidden" name="crumb" value="{{crumb}}"/>
+                    <input type="hidden" id="holdId" name="holdId" value="{{ paymentHold.holdId }}">
+                    <button class="govuk-button govuk-button--secondary" data-module="govuk-button" style="margin-bottom: 0px;">Remove</button>
```

Comment:
Clicking on `Remove` doesn't remove the `hold`.

The `hold` in `ffc-pay-processing` has been closed though

Reply:
Fixed.  I wasn't filtering out closed holds as I wrongly assumed the API I'm calling only returned open holds.  Maybe it should longer term or those payloads are going to be huge in time.

Reply:
Thinking about it, I'm definitely going to create a ticket or performance is going to suffer in a years time.